src/Tizen.Applications.Common/Tizen.Applications/ApplicationRunningContext.cs

```diff
@@ -255,6 +255,31 @@ public void Terminate()
             }
         }
 
+        /// <summary>
+        /// Terminates the background application.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed of invalid argument.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when failed because of permission denied.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of system error.</exception>
+        /// <privilege>http://tizen.org/privilege/appmanager.kill.bgapp</privilege>
+        /// <since_tizen> 6 </since_tizen>
+        public void TerminateBgApp()
```

Comment:
Why it added in ApplicationRunningContext class instead of ApplicationManager?

Reply:
What is different from `Terminate()`

Reply:
We intend to use ApplicationRunningContext as managing application's running context, so it contains Resume, Terminate methods.

And TerminateBgApp is terminate only background application as summary described

Reply:
As your comment
How about adding overloading method of Terminate()?

```suggestion
        public void Terminate(bool onlyForBackground);
```

Reply:
And need to update description. 

`Terminates the application if it is running on background`

Reply:
Terminate(false); case is internal API we do not have plan to open it public

Reply:
How about add in ApplicationManager class as static?

public static class ApplicationManager {
   public static void TerminateBackgroundApplication(ApplicationRunningContext app);
}

Reply:
Our design intend to managing app running context using ApplicationRunningContext.
So, Resume() and internal Terminate() methods also included in ApplicationRunningContext.
static TerminateBackgroundApplication method in ApplicationManager class is not consistent with our design.

Reply:
Terminate is acceptable for method of ApplicationRunningContext.

But, TerminateBackgroundApplication is not acceptable for ApplicationRunningContext

Reply:
as you say, Terminate is internal API and if we can change api signature, How about change behavior of Terminate() as terminate background app?
and orignal api is changed to Terminate(bool includeForground)